doc/org/build/networking.md

```diff
@@ -0,0 +1,256 @@
+
+
+<a id="org22fd151"></a>
+
+# Persisting Networked Entity State
+
+By default, `network instantiated` entities are removed when its `creator` (a client) disconnects. In order to persist these entities (and its descendants) the entity must be `pinned`. Only `network instantiated` entities can be `pinned`.
+
+To `pin` an `network instantiated` entity, a client calls `createEntityState` . This will save the current state of the entity (and its descendants) to the database. We say that the entity (and its descendants) are `persistent`.
+
+To update the state of a persistent entity, a client calls `updateEntityState`.
+
+To delete the saved entity state of a persistent entity, a client calls `deleteEntityState`. Note that deleting the saved entity state is not the same as deleting the entity. It simply means that the information saved to the database about this entity will be deleted.
+
+When the client connects to a hub channel, it calls `listEntityStates` in order to receive the entity states that have been saved to the database.
+
+Saved entity states include `CreateMessage` s for the `network instantiated` entity and `UpdateMessage` s for itself and its descendants. These messages are queued and later processed by the `networkReceiveSystem` like any other messages. The client&rsquo;s eventually consistent properties guarantee that if entity state updates that come from the database are out-of-date, they will be appropriately handled (i.e. ignored).
+
+
+<a id="org7edeaa3"></a>
+
+# Networked Entity Hierarchies
+
+When `createNetworkedEntity` is called, a `network instantiated` entity is created synchronously. That is, any asynchronous loading that an entity needs to do to be &ldquo;fully realized&rdquo; will happen later.
+
+For example, consider a call to `createNetworkedEntity(world, "media", params)` . The media prefab&rsquo;s template function (shown in a previous section) will cause an entity with a `MediaLoader` to be created.
+
+If `params.src` point at a URL where a GLB model file is hosted, then the model will be downloaded, loaded by the THREE GLTFLoader, its nested components will be inflated by inflators, and finally those `object3D` s will be added to the scene graph.
+
+Between the time that the `network instantiated` entity is created (e.g. upon receiving a `CreateMessage`) and the time that the descendant entities are created (with associated `object3D` s), clients may receive `UpdateMessage` s about descendant entities it does not recognize.
+
+Clients store these `UpdateMessage` s until they can be applied.
+
+A critical property of the networked system that enables this to work is that descendants of `networked instantiated` entities are assigned network IDs deterministically, even in cases where some parts of a descendant hierarchy fails to load. This ensures that the descendants can load in any order (or even fail to load) without causing a client to delete, overwrite, or ignore descendant updates.
+
```

Comment:
LAST NOTE: This documentation is amazing and I have 3 overall suggestions...
1. Explain further now bitECS pairs with THREE.js entities. As I said in my comment above, beginner developers and hackers like me will be looking for something familiar once the full transition is complete and THREE is something most will likely have some familiarity with.

2. Keep adding examples that explain the small things quirks of bitECS, like you did with the troika text example. 

3. Broadcast the heck out of this documentation and put it links to it outside of the github